intellst-api/src/DTO/UserDTO.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\DTO;
+
+use JMS\Serializer\Annotation as Serializer;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class UserDTO
+{
+     /**
+     * User ID
+     * @var integer
```

Comment:
we already have type in variable declaration, so it isn't needed here, same for other vars